File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_One/Java/Minimum.java
import java.io.*;


class Minimun{

	
	public static void main(String args[])throws IOException{

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		System.out.printf("Enter the number1 : ");
		int num1 = Integer.parseInt(br.readLine());
		System.out.printf("Enter the number2 : ");
		int num2 = Integer.parseInt(br.readLine());
			
		System.out.println("The Minimum number amongst"+num1+" and "+num2+" is "+(num1<num2?num1:num2));

	}	
}	   

File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_One/C/Minimum.c
#include<stdio.h>

void main(){

	int num1,num2;

	printf("Enter number 1: ");
	scanf("%d",&num1);

	printf("Enter number 2: ");
	scanf("%d",&num2);

	printf("The Minimun number amongst %d & %d is %d",num1,num2,num1<num2?num1:num2);

}	
